docs/rules/attributes-order.md

```diff
@@ -35,7 +35,11 @@ This rule aims to enforce ordering of component attributes. The default order is
 - `OTHER_DIRECTIVES`
   e.g. 'v-custom-directive'
 - `OTHER_ATTR`
-  e.g. 'custom-prop="foo"', 'v-bind:prop="foo"', ':prop="foo"'
+  alias for '[OTHER_ATTR_DYNAMIC, OTHER_ATTR_STATIC]'
+- `OTHER_ATTR_DYNAMIC`
+  e.g. 'v-bind:prop="foo"', ':prop="foo"'
+- `OTHER_ATTR_STATIC`
+  e.g. 'prop="foo"', 'custom-prop="foo"'
```

Comment:
Note that in this list, the default order is documented. So it doesn't make sense to list `OTHER_ATTR`, `OTHER_ATTR_DYNAMIC` and `OTHER_ATTR_STATIC` in the same level. I'd change it like this:

```suggestion
  alias for `[OTHER_ATTR_DYNAMIC, OTHER_ATTR_STATIC]`:
  - `OTHER_ATTR_DYNAMIC`
    e.g. 'v-bind:prop="foo"', ':prop="foo"'
  - `OTHER_ATTR_STATIC`
    e.g. 'prop="foo"', 'custom-prop="foo"'
```

Reply:
Fixed